src/clients/travis.js

```diff
@@ -73,7 +74,7 @@ export async function getUserInformation(travisAccessToken, account) {
  * @returns {Promise}
  */
 export function getRepositoryId(travisAccessToken, username, projectName) {
-  winston.log('verbose', 'getRepositoryId', { username, projectName });
+  winston.log('verbose', 'getting repository id from travis', { username, projectName });
```

Comment:
I don't know, I like the method name here better

Reply:
I don't know, maybe we can do something like: `functionName : description`.

But at the end of the day, these messages would go to the user too. So a function name might not be useful.

Reply:
I like the method name for debugging, but if we plan on giving the user a verbose command line option, it should be a phrase in standard English. So... maybe have two things that get logged: the method name at the `debug` level and the phrase at the `verbose` level?